.gitignore

```diff
@@ -5,10 +5,11 @@
 /.vscode/
 /htmlcov/
 /logs
-/mails/
+mails/
 /media/
 /reports/
 /static/
+venv/
```

Comment:
shouldn't be necessary when using pipenv

Reply:
Did not know pipenv beforehand so to get the project up and running I just used the (for me well known) python module 'venv'.